app/assets/javascripts/admin/addon/models/tl3-requirements.js

```diff
@@ -18,6 +18,7 @@ export default EmberObject.extend({
   },
 
   @discourseComputed(
+    "post_edits_count",
```

Comment:
Out of curiosity, why do we need to add this here?

Reply:
Leftover from doing some tests to get to know the codebase